subsys/bluetooth/controller/Kconfig

```diff
@@ -584,5 +584,13 @@ config BT_CTLR_SDC_CS_COUNT
 	help
 	  Set the number of concurrent connections that can support Channel Sounding procedure.
 
+config BT_CTLR_SDC_LE_POWER_CLASS_1
+	bool "Device supports transmitting at LE Power Class 1 level"
```

Comment:
Will you add a corresponding `config BT_CTLR_LE_POWER_CLASS_1_SUPPORT` upstream?

Reply:
I wasn't planning on it, do you think this is valuable?

Reply:
We do so for other LL features, so we should add it for consistency. Can be left out of the current PR